test/docker-compose.yml

```diff
@@ -19,7 +19,7 @@ services:
       - HUB_PORT_4444_TCP_PORT=4444
   tests:
     image: ${DOCKER_REGISTRY}node:12
-    command: bash -c "cp -R /data/. /workdir && npm install && npm run test:grid"
+    command: bash -c "cp -R /data/. /workdir && npm install && npm run test"
```

Comment:
Moet op de omgeving niet het grid script runnen?

Reply:
Yes, foutje, ik pas het aan. Mogelijks fout in de util en is dan overschreven.